assets/js/aplus.js

```diff
@@ -86,6 +86,37 @@ $(function() {
 
     $(window).bind('scroll', modifyMenu);
     $(window).bind('resize', updateMenu);
+
+    /**
+    * Warn about links that open in new windows.
+    */
+
+    function addLinkType(link, type) {
+      let linkTypes = (link.getAttribute('rel') || '').split(' ');
+      if (!linkTypes.includes(type)) {
+        linkTypes.push(type);
+      }
+      link.setAttribute('rel', linkTypes.join(' ').trim());
+    }
+
+    function addExternalLinkIcon(link) {
+      if (!link.querySelector('.icon')) {
+        link.insertAdjacentHTML('beforeend', `<span class="icon external-link"></span>`);
+      }
+    }
+
+    function addScreenReaderMessage(link, message) {
+      if (!link.querySelector('.screen-reader-only')) {
+        link.insertAdjacentHTML('beforeend', `<span class="screen-reader-only"> (${message})</span>`);
+      }
+    }
+
+    document.querySelectorAll('a[target="_blank"]').forEach(link => {
+      console.log("open in a new tab");
+      addLinkType(link, 'noopener');
+      addExternalLinkIcon(link)
+      addScreenReaderMessage(link, 'opens in a new tab');
```

Comment:
The message `opens in a new tab` must be translated (in Finnish it is `avautuu uudessa välilehdessä`).